src/res_comments_table.ml

```diff
@@ -692,9 +692,12 @@ and walkTypeDeclaration (td : Parsetree.type_declaration) t comments =
     | Ptype_abstract | Ptype_open -> rest
     | Ptype_record labelDeclarations ->
       let () =
-        walkList
-          (labelDeclarations |> List.map (fun ld -> LabelDeclaration ld))
-          t rest
+        if List.length labelDeclarations = 0 then
```

Comment:
`List.length` iterates over the list unnecessarily, please use `match labelDeclarations with [] -> ...` instead.